hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithFSO.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.hadoop.hdds.scm.container.ContainerNotFoundException;
 import org.apache.hadoop.hdds.scm.container.ContainerReplica;
 import org.apache.hadoop.hdds.scm.server.OzoneStorageContainerManager;
+import org.apache.hadoop.hdds.scm.container.placement.metrics.SCMNodeStat;
```

Comment:
It seems this import is missing from `TestNSSummaryEndpointWithLegacy`, causing build failure.

https://github.com/apache/ozone/actions/runs/3481884746/jobs/5823544988#step:6:2109

Reply:
Thanks Atila for the quick review! updated!